cli/lsp/client.rs

```diff
@@ -259,6 +276,18 @@ impl ClientTrait for TowerClient {
     }
   }
 
+  async fn send_did_change_deno_configuration_notification(
```

Comment:
Out of curiosity - why do we need to have two almost identical `send_did_change_deno_configuration_notification`? Is it something that could be refactored?

Reply:
They're for different trait implementations I guess. This is done for all our custom notifications. It's already factored out since it's a one-liner.